presentations/5-commitments.tex

```diff
@@ -0,0 +1,268 @@
+\documentclass[xcolor={usenames,dvipsnames}]{beamer}
```

Comment:
Presentation should have the same fixes as the lecture notes file:
1. Instead of $\mathbb{C}$, use $\mathcal{C}$ for the challenge space.
2. Commitments are denoted by $c$, not by $\mathcal{C}$.
3. Do not use $\mathbb{Z}$, but a subspace of it.
4. Etc.